src/components/navBar/navBar.module.scss

```diff
@@ -1,14 +1,17 @@
 .navBar {
+  display: flex;
   width: 100%;
-  background-color: #1d1283;  
+  background-color: #1d1283;
+  height: 5.25rem;
```

Comment:
The height should not be mentioned ideally, it should adjust itself with the content.
The problem you are facing here is the underline below the anchor tag (`<a>`)  looks out of the `navbar`.
To solve this problem, you can add a `height` to your `<a>` which needs to be greater than the `underline-offset` and `decoration-thickness`.

Let me know if you don't understand the problem or the solution 😄